notebooks/Analyze_Text.ipynb

```diff
@@ -0,0 +1,2895 @@
+{
```

Comment:
<p>It would be good to have a sentence describe what kind of data is in each of the following DataFrames, I could do that as a followup.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>Good point. I've created issue #101 to cover creating this additional text.</p>